### PR TITLE
ENG-2696: Fix restore seed phrase screen scroll

### PR DESCRIPTION
**Pull request by jordankzf**
# 🔘 PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Enhancement
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

# 📜 Background
Provide a brief explanation of why this pull request is needed. Include the problem you are solving or the functionality you are adding. Reference any related issues.

Issue Link: #[issue_number]
Context Link (if applicable):

# 🔄 Changes
Enumerate the changes made in this pull request, detailing what has been modified, added, or removed. Include technical details and implications if necessary.

Impact:
- Explain the broader impact of these changes.
- How it improves performance, fixes bugs, adds functionality, etc.

# 🖼 Screenshot / 📹 Video
Include screenshots or a video demonstrating the changes. This is especially helpful for UI changes.

# ✅ Review checklist
Please ensure the following are true before merging:

- [ ] Code Style is consistent with the project guidelines.
- [ ] Code is readable and well-commented.
- [ ] No unnecessary or debugging code has been added.
- [ ] Security considerations have been taken into account.
- [ ] The change has been manually tested and works as expected.
- [ ] Breaking changes and their impacts have been considered and documented.
- [ ] Code does not introduce new technical debt or issues.

### CODE REVIEW

**Comment by jordankzf** [comment]
Fixed @DuskaT021's reported issue. Thanks!

**Review by teebszet — APPROVED** [review]

File: src/pages/Options/index.css [hunk 1]
```diff
@@ -2,8 +2,9 @@ body {
   margin: 0;
   padding: 0;
   color: #ffffff;
-  background-color: #0C0C0C;
+  background-color: #0c0c0c;
   position: relative;
+  overflow-x: hidden;
```
Comment (teebszet) [inline src/pages/Options/index.css:7]: I'm curious what did this solve for?

File: src/pages/Options/index.css [hunk 2]
```diff
@@ -22,7 +23,8 @@ header h2 {
 }
 
 #app-container {
-  height: 100vh;
+  height: 100%;
+  min-height: 100vh;
```
Comment (teebszet) [inline src/pages/Options/index.css:27]: did you check that this doesn't break the other screens which open up in via options.html?
e.g. any of the collectibles screens when you click 'Open gallery' button, or the ledger send flows

File: src/app/components/extendedScreenContainer/index.tsx [hunk 3]
```diff
@@ -6,11 +6,13 @@ import styled from 'styled-components';
 const ExtendedScreenRouteContainer = styled.div((props) => ({
   display: 'flex',
   flexDirection: 'column',
-  height: '100vh',
+  height: '100%',
+  minHeight: '100vh',
```
Comment (teebszet) [inline src/app/components/extendedScreenContainer/index.tsx:10]: same question as below. 

did you check that this doesn't break the other screens which open up via options.html?
e.g. any of the collectibles screens when you click 'Open gallery' button, or the ledger send flows

File: src/app/components/extendedScreenContainer/index.tsx [hunk 4]
```diff
@@ -6,11 +6,13 @@ import styled from 'styled-components';
...
   width: '100vw',
   backgroundColor: props.theme.colors.elevation0,
   border: `1px solid ${props.theme.colors.elevation2}`,
   boxShadow: '0px 8px 28px rgba(0, 0, 0, 0.35)',
+  overflow: 'hidden',
```
Comment (teebszet) [inline src/app/components/extendedScreenContainer/index.tsx:15]: I'm curious what did this solve for?

**Comment by teebszet** [comment]
@jordankzf even though this is a minor UI fix, I think when you touch the css/layout which is shared across many screens, it requires a thorough check that nothing was broken. 

in this case, it's a good idea to at least include screenshots or video of the affected layouts behaving as expected compared to the previous layouts

File: src/pages/Options/index.css (hunk 1 above)
Comment (jordankzf) [reply src/pages/Options/index.css:7]: [scrnli_1_11_2024_2-24-51 PM.webm](https://github.com/secretkeylabs/xverse-web-extension/assets/65149726/f9170b5c-e65f-49b5-8d74-ec3d86da5c7f)

`overflow-x: hidden;` removes the horizontal scrollbar here

File: src/app/components/extendedScreenContainer/index.tsx (hunk 4 above)
Comment (jordankzf) [reply src/app/components/extendedScreenContainer/index.tsx:15]: Seems like this can be safely removed.

**Comment by jordankzf** [comment]
image

Ledger layout unaffected

image
image
Collectibles layout unaffected

File: src/pages/Options/index.css (hunk 2 above)
Comment (jordankzf) [reply src/pages/Options/index.css:27]: Yes, I did! Attached screenshots to a new comment.

**Pull request merged**